x/crosschain/keeper/msg_server_migrate_tss_funds_test.go

```diff
@@ -35,20 +35,38 @@ func TestMsgServer_MigrateTssFunds(t *testing.T) {
 		_, found := k.GetCrossChainTx(ctx, index)
 		assert.True(t, found)
 	})
+	t.Run("not enough funds in tss address for migration", func(t *testing.T) {
+		k, ctx, _, zk := keepertest.CrosschainKeeper(t)
+		admin := sample.AccAddress()
+		setAdminPolicies(ctx, zk, admin)
+		msgServer := keeper.NewMsgServerImpl(*k)
+		chain := getValidEthChain(t)
+		amount := sdkmath.NewUintFromString("100")
+		indexString, _ := setupTssMigrationParams(zk, k, ctx, *chain, amount, true, true)
+		_, err := msgServer.MigrateTssFunds(ctx, &crosschaintypes.MsgMigrateTssFunds{
+			Creator: admin,
+			ChainId: chain.ChainId,
+			Amount:  amount,
+		})
+		assert.ErrorContains(t, err, crosschaintypes.ErrCannotMigrateTssFunds.Error())
```

Comment:
Can we replace all `assert` with `require` in this file? We use it everywhere else and it makes easier for troubleshooting the tests

Reply:
1a24f7336d587480ed91c3bb65541a2fdecc01ae